lib/project_types/script/layers/application/create_script.rb

```diff
@@ -19,16 +26,36 @@ def call(ctx:, language:, script_name:, extension_point_type:, description:)
 
           private
 
-          def setup_project(ctx, language, script_name, extension_point, description)
+          DEFAULT_CONFIG_UI_FILENAME = "ui-config.yml"
+          DEFAULT_CONFIG = {
+            "version" => 1,
+            "type" => "single",
+            "fields" => [],
+          }
```

Comment:
I think eventually, each default Script would have a corresponding `ui-config.yml` file there.  So, we won't have to do this.  Maybe the EP authoring process should guarantee that this file exists?  WDYT?  🤔

Reply:
Agreed. If this is the default, once we move to sparse-checkout and self-contained samples, we should add this config file directly to the script and only modify it in the creation process if need be. Should we create an issue to track this @jacobsteves? Alternatively, we could add this as part of the acceptance criteria for the sparse-checkout ticket.

Reply:
> once we move to sparse-checkout and self-contained samples
We don't need to wait, but it will be easier to do so once we start using sparse checkout, I'd argue that adding sparse checkout is better than adding more files.

Reply:
I am ok with the your suggested sequencing.  From the tutorial pov (+CC @kwringe), I think it will be really like to have the hello-world example for each EP, such that the Script author just bootstraps the default Script from CLI, push, install on dev shop, and see everything working e2e with the config UI in the Admin.  The Script author can the just configure from the Admin and see the Scripts working in the dev shop.  After the API team updates the payment/shipping APIs, maybe we can think about what needs to be done in order to make the hello-world example easier to explain in the tutorial.  Maybe it's:

- Sparse checkout
- Update each sample and example Scripts with the right ui-config.yml

WDYT?

Reply:
Sounds good to me.

Reply:
I agree! I opened an issue to track it https://github.com/Shopify/script-service/issues/2696